templates/compare/index.html

```diff
@@ -0,0 +1,289 @@
+{% extends "base_layout.html" %}
+
+{% block title %}MicroK8s vs k3s vs Minikube{% endblock %}
+
+{% block meta_description %}Feature comparison of lightweight Kubernetes distributions: MicroK8s, K3s and minikube.{% endblock meta_description %}
+
+{% block meta_image%}https://assets.ubuntu.com/v1/920467d2-Compare_MicroK8s-K3S-Minikube.svg{% endblock meta_image%}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1wWos3iWMTCqDbxZBDuSVqvY6ZwYux-goCrX0L0E5Weo/edit{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>MicroK8s vs K3s vs minikube</h1>
+      <p>Lightweight Kubernetes distributions are becoming increasingly popular for local development, edge/IoT container management and self-contained application deployments. If you are looking to run Kubernetes on devices lighter in resources, have a look at the table below. MicroK8s stands out for its simplicity, robustness and completeness.</p> 
+      <a class="p-button--positive js-invoke-modal" href="/contact-us">Get in touch</a>
+    </div>
+    <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
+      <!-- This image needs to be updated -->
+      <img src="https://assets.ubuntu.com/v1/920467d2-Compare_MicroK8s-K3S-Minikube.svg" alt="" width="240" height="150" loading="lazy" />
+    </div>
+  </div>
+</section>
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <table class="p-table--mobile-card">
+      <thead>
+        <tr>
+          <th>&nbsp;</th>
+          <th class="u-align--center">MicroK8s</th>
+          <th class="u-align--center">K3s</th>
+          <th class="u-align--center">minikube</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td aria-label="Category">CNCF certified</td>
+          <td aria-label="MicroK8s" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
```

Comment:
I'm pretty sure we decided to start phasing out the aubergine tick icon, in favour of the green one available in Vanilla. See the table on https://ubuntu.com/security/cis, under the heading "Which versions of Ubuntu have CIS tooling?":

```suggestion
            <i class="p-icon--success">Yes</i>
```